transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/SelectTransferMethodPresenter.java

```diff
@@ -268,14 +267,17 @@ public void onKeysLoaded(@Nullable HyperwalletTransferMethodConfigurationKey key
                 if (!mView.isActive()) {
                     return;
                 }
+                String selectedDefaultCurrencyCode = getDefaultCurrencyCode(key, countryCode);
```

Comment:
I checked  the code and I don't think we need to change this method.  The `currencyCode` will be informed form the UI 

https://github.com/hyperwallet/hyperwallet-android-ui-sdk/blob/5dbafff880815b8b6f64b2e6ae6e0d171eb3e6f6/transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/SelectTransferMethodFragment.java#L394